doc/21-development.md

```diff
@@ -848,7 +848,7 @@ current year as this implies yearly updates we don't want.
 
 Depending on the file type, this must be a comment.
 
-```
+```cpp
```

Comment:
The whole block shouldn't be C++. Split the block into two and use another language where `#` is actually a comment. Or is there a language where both `/* */` and `#` are comments?